File: conditionals/time.py
import datetime

now=datetime.datetime.now().hour-6

print(now)

if now <= 12:
    print("Good Morning!\n")
elif now <= 18:
    print("Good afternoon!\n")
elif now <= 20:
    print("Good evening!\n")
else:
    print("Good night! GO to bed!")

File: conditionals/time.c
#include <stdio.h>
#include <time.h>

int hour;


int main(void){
    time_t now;
    struct tm*now_tm;

    now = time(NULL);
    now_tm = localtime(&now);
    hour = (now_tm->tm_hour)-6;
    printf("%d\n", hour);

    if (hour <= 12){
        printf("Good Mornings.\n");
    }else if (hour <= 18){
        printf("Good Afternoon.\n");
    }else if (hour <= 20){
        printf("Good Evening\n");
    }else{
        printf("Good night! GO TO BED!\n");
    }

    return 0;
}